content/blog/2024-06-19-ethereum-classic-vs-central-bank-digital-currencies-CBDC-s/index.md

```diff
@@ -0,0 +1,166 @@
+---
+title: "Ethereum Classic vs Central Bank Digital Currencies (CBDCs)"
+date: 2024-06-19
+author: Donald McIntyre
+contributors: ["DonaldMcIntyre"]
+tags: ["philosophy"]
+linkImage: ./banner.png
+---
+
+---
+**You can listen to or watch this video here:**
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/IDr_mPliX70" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
+
+---
+
+![](./banner.png)
+
+As central banks all over the world race to launch their Central Bank Digital Currencies (CBDCs) to please the global elites such as the [World Economic Forum](https://www.weforum.org/publications/modernizing-financial-markets-with-wcbdc/), we as citizens have no other recourse but to learn and understand what these new technologies will entail to our life, liberty, and property.
+
+To explain what are CBDCs, a good way of going about it is to compare them with Ethereum Classic (ETC).
+
+ETC is the most secure and advanced blockchain technology in the world and CBDCs seek to imitate its design but to add to them their central control and planning devices.
+
+Bitcoin was not a useful model to follow by them as it was so basic that it didn’t enable any kind of central planning scheme.
+
+![](./1.png)
+
+However, the EVM model invented by Vitalik Buterin, which is the design that ETC follows, is very versatile and is only truly decentralized when it works with the proof of work based Nakamoto Consensus mechanism.
+
+Not even Ethereum (ETH) is decentralized today. 
+
+ETH decided to migrate from POW to proof of stake (POS) in 2022, which converted it into a hybrid between ETC and CBDCs. 
+
+However, this hybrid, as it doesn’t use any work to achieve decentralization, is easily capturable by government and special interests as CBDCs already are by definition and construction.
+
+In the next sections we will make a comparison of ETC with CBDCs to illustrate the incredible similarities of these systems, but at the same time the critical differences between them which basically mean the difference between freedom or tyranny.
+
+## Comparison Table
+
+![](./2.png)
+
+The table above lists the features we will compare between ETC and CBDCs, and we mark how they meet each specification. 
+
+Below we will explain the comparisons line by line.
+
+### Consensus Mechanism
+
+![](./3.png)
+
+As we said above, the only way to make a blockchain network decentralized is by using proof of work as the consensus mechanism. This reality produces three segments in the blockchain industry: POW decentralized blockchains such as Bitcoin and Ethereum Classic; POS or proof of authority (POA) networks such as Ethereum and Ripple; and strictly POA central planning government systems such as CBDCs.
+
```

Comment:
define POA.

Reply:
A Proof of Authority blockchain is not POW that uses miners nor POS that uses stakers. In POA, the block producers and validators are trusted entities that are selected to receive transactions, organize them, and update the ledger. They get a payment just like miners or stakers, but they are manually chosen trusted parties.

For example Ripple and EOS work like this (EOS is a mix of POS and POA - hence called delegated POS or DPOS - because the block producers also put up capital to gain that privilege).